API.md

```diff
@@ -813,6 +813,8 @@ If there was an error, expect an HTTP status code in either the 4XX or 5XX range
 
 Registers a new data object and/or replica into the catalog.
 
+This operation requires rodsadmin level privileges.
```

Comment:
This depends on the server configuration. From https://docs.irods.org/4.3.2/icommands/user/#ireg:
> By default, only rodsAdmin users are allowed to register files or directories.
More permissive registration policies can be set in the server's rule base
using the acNoChkFilePathPerm() rule.

Reply:
Will tweak.

Reply:
How about ...
```
This operation may require rodsadmin level privileges depending on the configuration
of the iRODS zone. Contact the administrator of the iRODS zone to be sure non-rodsadmin
users are allowed to execute this operation.
```

Reply:
That sounds good to me. Technically I guess it's on a per-server basis since it's a configured rule, but I don't think that needs to be disclosed here. :p

Reply:
done